src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2308,66 +2295,6 @@ MavlinkReceiver::handle_message_hil_state_quaternion(mavlink_message_t *msg)
 			orb_publish(ORB_ID(battery_status), _battery_pub, &hil_battery_status);
 		}
 	}
-
```

Comment:
This looks like it will break HIL - I don't see the equivalents of what has been removed here being published?